poppler: 22.06.0 -> 22.08.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/poppler-glib/versions.


meta.description for poppler is: "A PDF rendering library"

meta.homepage for poppler is: "https://poppler.freedesktop.org/"

meta.changelog for poppler is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 22.07.0 with grep in /nix/store/6f1csfw133f474slrpw87a4lcas4vq31-poppler-glib-22.07.0
- found 22.07.0 in filename of file in /nix/store/6f1csfw133f474slrpw87a4lcas4vq31-poppler-glib-22.07.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
634 total rebuild path(s)

634 package rebuild(s)

First fifty rebuilds by attrpath
CuboCore.corepdf
adapta-gtk-theme
anystyle-cli
apacheHttpdPackages.mod_tile
apacheHttpdPackages_2_4.mod_tile
apvlv
arc-theme
arx-libertatis
asciidoc-full
asciidoc-full-with-plugins
auto-multiple-choice
ayu-theme-gtk
bookworm
calibre
calligra
capitaine-cursors
chrome-gnome-shell
cinnamon.cinnamon-common
cinnamon.cinnamon-control-center
cinnamon.cinnamon-gsettings-overrides
cinnamon.cinnamon-screensaver
cinnamon.cinnamon-session
cinnamon.cinnamon-settings-daemon
cinnamon.iso-flags-png-320x420
cinnamon.iso-flags-svg
cinnamon.xreader
claws-mail
clevis
cloudcompare
colord
colord-gtk
colord-gtk4
cups-filters
darktable
dblatexFull
denemo
diff-pdf
diffoscope
diffpdf
digikam
disorderfs
dropbox-cli
eaglemode
eiciel
emojione
engauge-digitizer
enlightenment.econnman
enlightenment.ecrire
enlightenment.efl
enlightenment.enlightenment
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/6f1csfw133f474slrpw87a4lcas4vq31-poppler-glib-22.07.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A poppler https://github.com/r-ryantm/nixpkgs/archive/f91849f9c35465af5375559bdbb5d14a2c2a61b6.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/6f1csfw133f474slrpw87a4lcas4vq31-poppler-glib-22.07.0
ls -la /nix/store/6f1csfw133f474slrpw87a4lcas4vq31-poppler-glib-22.07.0/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @ttuegel @jtojnar for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

anthonyroussel commented:
> Release 22.07.0:
>
> 
>         core:
>          * Fix crash when filling in forms in some files. Issue #1258
>          * Fix first lines of Annotations sometimes being cut off. Issue #1246
>          * Signatures: Don't crash if the signature doesn't have a common name
>          * CairoFontEngine: increment font_face reference when retrieving from the cache
>          * Add ToUnicode support for lessorequalslant and greaterorequalslant
> 
>         glib:
>          * Add support for stamp annotation
> 
>         build system:
>          * Tweaks on how gperf is run

See https://poppler.freedesktop.org/releases.html

jtojnar committed:
poppler: 22.06.0 → 22.08.0

https://lists.freedesktop.org/archives/poppler/2022-July/015263.html
https://lists.freedesktop.org/archives/poppler/2022-August/015278.html

jtojnar reviewed (approved):
`poppler.tests` pass on x86_64-linux.
